engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -615,8 +624,7 @@ public Camera getLightCamera() {
 
     @Override
     public void propertyChange(PropertyChangeEvent evt) {
-        renderPipelineTaskList = generator.regenerate();
-        logger.info("Set {} property to {}. ", evt.getPropertyName().toUpperCase(), evt.getNewValue());
+        logger.info("Set {} property to {}. ", evt.getPropertyName().toUpperCase(), evt.getNewValue()); // for debugging purposes
```

Comment:
Hmmmm.... how about you move this logging functionality into the RenderingDebugConfig? You could perhaps make the RenderingDebugConfig subscribe to itself and reuse this method as is.